src/elements/content-sidebar/SidebarNav.js

```diff
@@ -46,52 +47,70 @@ const SidebarNav = ({
     isOpen,
     onNavigate,
 }: Props) => (
-    <div aria-label={intl.formatMessage(messages.sidebarNavLabel)} role="tablist">
-        {hasActivity && (
-            <SidebarNavButton
-                interactionTarget={SIDEBAR_NAV_TARGETS.ACTIVITY}
+    <div className="bcs-SidebarNav" aria-label={intl.formatMessage(messages.sidebarNavLabel)} role="tablist">
+        <div className="bcs-SidebarNav-tabs">
+            {hasActivity && (
+                <SidebarNavButton
+                    data-resin-target={SIDEBAR_NAV_TARGETS.ACTIVITY}
```

Comment:
Decoupled the `data-testid`'s from the constants used for the `data-resin-target`'s